source/jormungandr/jormungandr/instance_manager.py

```diff
@@ -121,6 +121,7 @@ def register_instance(self, config):
             self._streetnetwork_backend_manager,
             config.get('external_services_providers', []),
             config.get('pt_planners', {}),
+            config.get('pt_journey_fare', {}),
```

Comment:
Not sure, for consistency, as the surrounding are plural:
```suggestion
            config.get('pt_journey_fares', {}),
```